src/examples/example_client.cpp

```diff
@@ -63,16 +63,22 @@ int main(int argc, char** argv)
       std::cout << "    " << d << std::endl;
 
     OpcUa::Node myvar;
+    OpcUa::Node mymethod;
 
     //Initialize Node myvar:
 
     //Get namespace index we are interested in
 
     // From freeOpcUa Server:
-    //uint32_t idx = client.GetNamespaceIndex("http://examples.freeopcua.github.io");
+    uint32_t idx = client.GetNamespaceIndex("http://examples.freeopcua.github.io");
     ////Get Node using path (BrowsePathToNodeId call)
     //std::vector<std::string> varpath({ std::to_string(idx) + ":NewObject", "MyVariable" });
     //myvar = objects.GetChild(varpath);
+    std::vector<std::string> methodpath({ std::to_string(idx) + ":NewObject", "MyMethod" });
+    mymethod = objects.GetChild(methodpath);
+    std::vector<OpcUa::Variant> arguments;
+    arguments.push_back(static_cast<uint8_t>(0));
+    mymethod.CallMethod(mymethod.GetId(), arguments);
```

Comment:
Maybe I misunderstand, but as far as I remember a method should always be called in the context of its parent node. so here the method should be called on the parent. It makes sense since the method often appies to the parent object.